thousandeyes/resource_stream.go

```diff
@@ -0,0 +1,217 @@
+package thousandeyes
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/thousandeyes/thousandeyes-sdk-go/v2"
+)
+
+type Stream struct {
+	ID                string            `json:"id,omitempty"`
+	Enabled           bool              `json:"enabled,omitempty"`
+	Type              string            `json:"type,omitempty"`
+	EndpointType      string            `json:"endpointType,omitempty"`
+	StreamEndpointUrl string            `json:"streamEndpointUrl,omitempty"`
+	DataModelVersion  string            `json:"dataModelVersion,omitempty"`
+	TestMatch         []StreamTestMatch `json:"testMatch,omitempty"`
+	TagMatch          []StreamTagMatch  `json:"tagMatch,omitempty"`
+}
```

Comment:
Was it intentional to omit these fields:
- `filters`
- `exporterConfig`
- `customHeaders`